### ISSUE TITLE
Need to add ? for Null Safe

### ISSUE THREAD

**Issue by sivaLeo** [Suraj-Tiwari/react-native-payumoney#51]
I'm getting error like
Type inference failed. Expected type mismatch: inferred type is ResultModel? but ResultModel was expected
Problem
   Inference
Solution
 Please add ? in 

> PayumoneyActivity.kt line:54:57  & 55:41

ex:
```java
       val transactionResponse: TransactionResponse? = data.getParcelableExtra(PayUmoneyFlowManager.INTENT_EXTRA_TRANSACTION_RESPONSE)
            val resultModel: ResultModel? = data.getParcelableExtra(PayUmoneyFlowManager.ARG_RESULT)

```

**Comment by sivaLeo** [comment]
i know its temporary solution. Any other solution please mention me....

**Comment by Suraj-Tiwari** [comment]
Maybe that's because underlying sdk update, I'll have a look at it

**Comment by sivaLeo** [comment]
Hi @Suraj-Tiwari , 
I still got that problem. Each time I work with the node_modules folder within your package. Any updates ??

**Comment by Suraj-Tiwari** [comment]
sorry the reason it had issue is, that file was created for testing and was removed, but i forgot to release, 
lemme quickly do one, 
actual fix is to delete that file altogether.

**Comment by Suraj-Tiwari** [comment]
released version 1.0.1 
reopen if issue exists
